apps/internal/migrations/0025_auto_generated_sql.py

```
@@ -0,0 +1,9 @@
+# Generated by Django
+from django.db import migrations
+from apps.internal.helpers import safe_run_sql
+sql_files = [
+    'fyle-integrations-db-migrations/intacct/functions/delete_workspace.sql'
+]
+class Migration(migrations.Migration):
+    dependencies = [('internal', '0024_auto_generated_sql')]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)